templates/quickstart-jira-dc.template.yaml

```diff
@@ -1358,9 +1360,9 @@ Outputs:
     Value: !If
       - UseCustomDnsName
       - !Sub
-        - "${HTTP}://${CustomDNSName}${ContextPath}"
+        - "${HTTP}://${CustomDomain}${ContextPath}"
```

Comment:
CustomDomain doesn't exist?

Reply:
CustomDomain comes from the definition on line 1365. This is part of the Sub command syntax I believe https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html